keylime/cloud_verifier_tornado.py

```diff
@@ -137,10 +146,10 @@ def store_attestation_state(agentAttestState):
             try:
                 session.add(update_agent)
             except SQLAlchemyError as e:
-                logger.error("SQLAlchemy Error on storing attestation state for agent %s: %s", agent_id, e)
+                logger.error("SQLAlchemy Error on storing attestation state: %s", e)
```

Comment:
This extra info (agent ID) was recently added by #1090. Is it intentional that you are removing it here? This also happens in other instances further on.

Reply:
Very strange, this doesn't seem intentional... I might have messed up my rebase.